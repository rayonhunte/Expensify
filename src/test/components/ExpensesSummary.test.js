import React from 'react';
import {shallow} from 'enzyme';
import ExpenseSummary, { ExpensesSummary } from '../../components/ExpensesSummary.js';



test('should render without expenses',()=>{
  const wrapper = shallow(<ExpensesSummary />);
  expect(wrapper).toMatchSnapshot();
});



test('should render with two expenses', ()=>{
  const wrapper = shallow(<ExpensesSummary expensesCount={2} expensesTotal={94.34}/>);
  expect(wrapper).toMatchSnapshot();
});


test('should render with one expenses', ()=>{
  const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={94.34}/>);
  expect(wrapper).toMatchSnapshot();
});