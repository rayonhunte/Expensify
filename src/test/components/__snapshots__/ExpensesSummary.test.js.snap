// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with one expenses 1`] = `
<h1>
  Viewing 
  1
   
  expense
  , Total 
  $0.9
</h1>
`;

exports[`should render with two expenses 1`] = `
<h1>
  Viewing 
  2
   
  expenses
  , Total 
  $0.9
</h1>
`;

exports[`should render without expenses 1`] = `
<h1>
  Viewing 
   
  expenses
  , Total 
  $0.0
</h1>
`;
